test/controllers/thesis_controller_test.rb

```diff
@@ -11,4 +11,23 @@ class ThesisControllerTest < ActionDispatch::IntegrationTest
     get '/thesis/new'
     assert_response :success
   end
+
+  test 'redirect after successful submission' do
+    sign_in users(:yo)
+    bernanke_title = "Long-term commitments, dynamic optimization, and the business cycle"
+    post '/thesis',
+      params: { thesis: {
+        title: bernanke_title,
+        abstract: "This thesis paper consists of three loosely connected essays. Each paper is a theoretical study of some form of long-term commitment made by economic agents.",
+        department_ids: [departments(:one).id.to_s],
+        degree_ids: [degrees(:one).id.to_s],
+        right_id: rights(:one).id.to_s,
+        graduation_year: (Time.current.year + 1).to_s,
+        graduation_month: 'December' } }
+    assert_response :redirect
+    assert_redirected_to thesis_path(Thesis.last)
+
```

Comment:
I think we need a follow_redirect in here to actually get the coverage

Reply:
Test passes.